front/src/app/app.module.ts

```diff
@@ -3,14 +3,21 @@ import { BrowserModule } from '@angular/platform-browser';
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
+import { StyleguideComponent } from './styleguide/styleguide.component';
+import {MatButtonModule} from '@angular/material/button';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+
 
 @NgModule({
   declarations: [
-    AppComponent
+    AppComponent,
+    StyleguideComponent
   ],
```

Comment:
This component should be displayed on specific route only + use lazy loading instead of  declaring component right here